simulation/traffic_simulator/include/traffic_simulator/api/api.hpp

```diff
@@ -135,11 +128,18 @@ class API
     return register_to_entity_manager() and register_to_environment_simulator();
   }
 
+  geometry_msgs::msg::Pose toMapPose(const geometry_msgs::msg::Pose & pose) { return pose; }
+
+  geometry_msgs::msg::Pose toMapPose(const traffic_simulator_msgs::msg::LaneletPose & pose)
+  {
+    return entity_manager_ptr_->getHdmapUtils()->toMapPose(pose).pose;
+  }
+
   template <typename Pose>
   auto spawn(
     const std::string & name, const Pose & pose,
     const traffic_simulator_msgs::msg::PedestrianParameters & parameters,
-    const std::string & behavior = PedestrianBehavior::defaultBehavior())
+    const std::string & behavior = PedestrianBehavior::defaultBehavior(), std::string model3d = "")
```

Comment:
```suggestion
    const std::string & behavior = PedestrianBehavior::defaultBehavior(), const std::string & model3d = "")
```

Reply:
The same reason as explained in [the comment](https://github.com/tier4/scenario_simulator_v2/pull/1028#discussion_r1268268536)

If requested, I can add the `model3d` to the `PedestrianParameters`.